eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -152,6 +153,11 @@ data ConwayGovPredFailure era
       ProtVer
       -- | The ProtVer of the Previous GovAction pointed to by the one proposed
       ProtVer
+  | InvalidPolicyHash
```

Comment:
Are these two policies hash related by anything other than being policy hashes? Otherwise why have a single predicate failure for both?